src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -67,6 +69,7 @@ public class FlowStateAllowedEventService {
                       REFER_TO_JUDGE,
                       REFER_TO_LEGAL_ADVISOR,
                       MAKE_DECISION,
+                      HEARING_SCHEDULED_GA,
```

Comment:
Just wondering, Does this should be allowed if even payment is failed in previous steps? Although   "HEARING_SCHEDULED_GA" has no fee included.

Reply:
Yes, this will be allowed after judge makes decision to keep the case as list for hearing